cfgov/v1/wagtail_hooks.py

```diff
@@ -30,7 +39,8 @@ def share_the_page(request, page):
 
     share(page, is_sharing, is_live)
     configure_page_revision(page, is_sharing, is_live)
-    flush_akamai(page, is_live)
+    if is_live:
+        flush_akamai(page)
```

Comment:
Should we check the return value here and do something if the flush failed? At a minimum a `log.error`?